var/spack/repos/builtin/packages/py-jupyter-client/package.py

```diff
@@ -25,3 +26,4 @@ class PyJupyterClient(PythonPackage):
     depends_on('py-traitlets', type=('build', 'run'))
     depends_on('py-jupyter-core', type=('build', 'run'))
     depends_on('py-zmq@13:', type=('build', 'run'))
```

Comment:
#13411 will rename this package to `py-pyzmq`, so one of us needs to remember to update this depending on which PR gets merged first. I have no personal preference.